README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/andela/ah-the-immortals-backend.svg?branch=develop)](https://travis-ci.org/andela/ah-the-immortals-backend)
```

Comment:
Hello @IssaIan ,
Would you make the badge to come after the title of the Project. 
The title gives a short phrase describing what the succeeding section of our project is all about, continuous integration being part of the project.

Reply:
Done.